src/components/DeeplinkWrapper/DeeplinkWrapper.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import {PureComponent} from 'react';
+
+const propTypes = {
+    /** Children to render. */
+    children: PropTypes.node.isRequired,
+};
+
+class DeeplinkWrapper extends PureComponent {
+    render() {
+        return this.props.children;
+    }
+}
+
+DeeplinkWrapper.propTypes = propTypes;
+export default DeeplinkWrapper;
```

Comment:
Can we just rename it to index.js & index.web.js? @cowboycito

Reply:
Done